### ISSUE TITLE
Getting IREEs pack based flow working to handle the OPT dispatches : GEMM

### ISSUE THREAD

**Issue by powderluv** [nod-ai/iree-amd-aie#150]

**Comment by MaheshRavishankar** [comment]
Separate issue to track status of compiling OPTs GEMM based dispatches (original collection of all dispatches https://github.com/nod-ai/iree-amd-aie/issues/96 ).

End state here should be that these dispatches should run end-to-end and tested on CI.

Blocked on #146 and on general codegeneration support built in `amd-aie` backend.
